scripts/sauce/sauce_connect_setup.sh

```diff
@@ -20,7 +19,15 @@ CONNECT_LOG="$LOGS_DIR/sauce-connect"
 CONNECT_STDOUT="$LOGS_DIR/sauce-connect.stdout"
 CONNECT_STDERR="$LOGS_DIR/sauce-connect.stderr"
 
-# Get Connect and start it
+# Get the appropriate URL for downloading Sauce Connect
+if [ `uname -s` = "Darwin" ]; then
```

Comment:
Yea, the issue with Windows support is that you have to handle each scenario separately.  For example, we would have to check for `cigwin` as well as any other terminal being used.  I say we open an issue for it and deal with it later.